code/web/src/modules/user/UserInfo.js

```diff
@@ -0,0 +1,105 @@
+// Imports
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+// UI Imports
+import { H3, H5 } from '../../ui/typography'
+import { Input, File, Textarea } from '../../ui/input'
+
+class UserInfo extends React.Component {
+  constructor() {
+    super() 
+    this.state = {
+      isEditing: false,
+      emailInput: '',
+      userDescription: '',
+      shippingAddressInput: ''
+    }
+  }
+
+  componentDidMount = () => {
```

Comment:
Sets local state on mounting of component (when the component renders to the page) using fetched data from the login network request.

Reply:
Thank You for inputting all of these notes, it is helpful when looking at the Delivery info file as well.